README.md

```diff
@@ -66,11 +66,11 @@ To use this Gem in 'production', we'll start by creating a PlanetScale Service T
 To configure your application in production, you'll need to feed it all of the right information via environment variables:
 
 ```
-PSCALE_ORG=<org_name>
-PSCALE_DB=<db_name>
-PSCALE_DB_BRANCH=main
-PSCALE_TOKEN_NAME=0sph6kvz5bxi
-PSCALE_TOKEN=<redacted>
+PLANETSCALE_ORG=<org_name>
+PLANETSCALE_DB=<db_name>
+PLANETSCALE_DB_BRANCH=main
+PLANETSCALE_TOKEN_NAME=0sph6kvz5bxi
+PLANETSCALE_TOKEN=<redacted>
```

Comment:
Should these be prefixed with `PLANETSCALE_SERVICE_` instead of `PLANETSCALE_` as the [docs suggest](https://docs.planetscale.com/tutorial/connect-any-application#authentication-in-production)?